### PR TITLE
[WIP] Chronos: new some deps in setup.py and dropped py36

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by TheaperDeng** [comment]
This PR will be postpone and be part of #4791

File: python/chronos/src/setup.py [hunk 1]
```diff
@@ -53,16 +53,17 @@ def setup_package():
...
         url='https://github.com/intel-analytics/BigDL',
         packages=get_bigdl_packages(),
-        install_requires=['bigdl-orca=='+VERSION, 'bigdl-nano[pytorch]',
-                          'pandas>=1.0.5, <1.3.0', 'scikit-learn', 'statsmodels',
-                          'onnx', 'onnxruntime'],
+        # neural_compressor, prophet, openvino-dev
+        install_requires=['bigdl-orca=='+VERSION, 'bigdl-nano=='+VERSION,
+                          'pandas>=1.0.5, <1.3.0', 'scikit-learn>=0.22.0, <=1.0.2', 'statsmodels',
+                          'onnx>=1.8.0, <=1.11.0', 'onnxruntime>=1.6.0, <=1.11.1', 'pmdarima~=1.8.4',
+                          'tsfresh==0.17.0', 'pyarrow~=6.0.1'],
```
Comment (TheaperDeng) [inline python/chronos/src/setup.py:60]: remove 'pmdarima~=1.8.4', 'tsfresh==0.17.0', 'pyarrow~=6.0.1' for now

Comment (liangs6212) [reply python/chronos/src/setup.py]: fixed

**Review by TheaperDeng — APPROVED** [review]

**Comment by liangs6212** [comment]
http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation/586/

**Pull request merged**